src/core/surface/call.c

```diff
@@ -539,12 +539,24 @@ grpc_compression_algorithm grpc_call_test_only_get_compression_algorithm(
   return algorithm;
 }
 
-static void set_encodings_accepted_by_peer(
-    grpc_call *call, const gpr_slice accept_encoding_slice) {
+static void destroy_encodings_accepted_by_peer(void *p) { return; }
```

Comment:
It looks like the changes in this file are not related to the changes in the rest of this PR - is that correct? I'm ok with leaving things in this PR, but I'd probably appreciate some commentary on the changes made here.